wled00/html_settings.h

```diff
@@ -187,7 +187,7 @@ const char PAGE_settings_sync[] PROGMEM = R"=====(<!DOCTYPE html>
 <h2>Sync setup</h2>
 <h3>Button setup</h3>
 On/Off button enabled: <input type="checkbox" name="BT"><br>
-Infrared receiver enabled: <input type="checkbox" name="IR"><br>
+Infrared receiver type (0 = disabled): <input name="IR" type="number" min="0" max="4" required><br>
```

Comment:
this changes the "ON/OFF" checkbox to a value entry field (0 to 4) with 0=disabled, 1=1st remote, 2=2nd remote etc. /// 4 different IR remotes in total.